stdlib/config-api/src/main/java/org/ballerinalang/stdlib/config/GetConfig.java

```diff
@@ -65,16 +65,27 @@ public static Object get(String configKey, String type) {
     @SuppressWarnings("unchecked")
     private static MapValue buildMapValue(Map<String, Object> section) {
         MapValue map = new MapValueImpl<String, Object>();
-
-        section.forEach((key, val) -> {
-            if (val instanceof String || val instanceof Long || val instanceof Double || val instanceof Boolean) {
-                map.put(key, val);
-            }
-        });
+        section.forEach((key, val) -> map.put(key, getConvertedValue(val)));
         return map;
     }
 
+    @SuppressWarnings("unchecked")
     private static BArray buildArrayValue(List value) {
-        return BValueCreator.createArrayValue(value.toArray(), new BArrayType(BTypes.typeAnydata));
+        Object[] convertedValues = new Object[value.size()];
+        value.forEach(convertedValue -> convertedValues[value.indexOf(convertedValue)] =
```

Comment:
same comment as above regarding the use of .forEach()

Reply:
Corrected.